cheatsheets/Transport_Layer_Security_Cheat_Sheet.md

```diff
@@ -136,19 +136,17 @@ For internal applications, an internal CA can be used. This means that the FQDN
+Certificates come in types of validation. Validation is the process the Certificate Authority uses to make sure you are allowed to have the certificate. This is authorization. The base validation is called Domain Validated (DV). All publicly issued certificates must be domain validated. This process involves practical proof of control of the name or endpoint requested in the certificate. This usually involves a challenge and response in DNS, to an official email address, or to the endpoint that will get the certificate.
 
-If the user does not know or trust this intermediate CA then the certificate validation will fail, even if the user trusts the ultimate root CA, as they cannot establish a chain of trust between the certificate and the root. In order to avoid this, any intermediate certificates should be provided alongside the main certificate.
+Organization Validated (OV) certificates include the subscriber’s organization information in the certificates subject DN. E.g. C = GB, ST = Manchester, **O = Sectigo Limited**, CN = sectigo.com. The process to acquire an OV certificate requires official contact with the subscriber company via a method that proves to the CA that they are truly talking to the right company.
 
-### Consider the use of Extended Validation Certificates
-
-Extended validation (EV) certificates claim to provide a higher level of verification of the entity, as they perform checks that the requestor is a legitimate legal entity, rather than just verifying the ownership of the domain name like normal (or "Domain Validated") certificates. This can effectively be viewed as the difference between "This site is really run by Example Company Inc." vs "This domain is really example.org".
+Extended validation (EV) certificates claim to provide an even higher level of verification of the entity than OV checks. They perform checks that the requestor is a legitimate legal entity, rather than just verifying the ownership of the domain name like OV certificates or simply showing practical proof of control, like DV certificates. This can effectively be viewed as the difference between "This site is really run by Example Company Inc." vs "This domain is really example.org".
 
 Historically these displayed differently in the browser, often showing the company name or a green icon or background in the address bar. However, as of 2019 both [Chrome](https://groups.google.com/a/chromium.org/forum/m/#!msg/security-dev/h1bTcoTpfeI/jUTk1z7VAAAJ) and [Firefox](https://groups.google.com/forum/m/?fromgroups&hl=en#!topic/firefox-dev/6wAg_PpnlY4) have announced that they will be removing these indicators, as they do not believe that EV certificates provide any additional protection.
```

Comment:
All major browsers have changed to not show any visual indicator to distinguish between validation types.